File: apps/connect4-api/src/main/kotlin/com/connect4/model/stats/RemainingTasks.kt
package com.connect4.model.stats

data class RemainingTasks (
  val threadInUse: Int = 0,
  val totalThread: Int = 0,
  val remainingBatches: Int = 0,
  val notStartedJobs: Int = 0,
  val runningJobs: Int = 0,
  val finishedJobs: Int = 0
)

File: apps/connect4/src/app/features/jobs/models/RemainingTasks.ts
export class RemainingTasks {
  constructor(
   public threadInUse: number,
   public totalThread: number,
   public remainingBatches: number,
   public remainingJobs: number,
   public notStartedJobs: number,
   public runningJobs: number,
   public finishedJobs: number,
  ) {}
}
